tests/unit/testExpressInterface.js

```diff
@@ -14,54 +14,42 @@
  * limitations under the License.
  */
 
-var test = require('tape');
 var lodash = require('lodash');
+var assert = require('assert');
 var merge = lodash.merge;
 var expressInterface = require('../../src/interfaces/express.js');
 var ErrorMessage = require('../../src/classes/error-message.js');
 var Fuzzer = require('../../utils/fuzzer.js');
 var Configuration = require('../fixtures/configuration.js');
 var createLogger = require('../../src/logger.js');
 
-test(
-  "Given invalid, variable input the express interface handler setup should not throw errors"
-  , function ( t ) {
-
-    var f = new Fuzzer();
-
-    t.doesNotThrow(
-      function ( ) {
-
-        f.fuzzFunctionForTypes(
-          expressInterface
-          , ["object", "object"]
+describe('expressInterface', function () {
+  describe('Exception handling', function () {
+    describe('Given invalid input', function () {
```

Comment:
Using describe like this to present hierarchy is an accepted part of mocha, and just because we don't have multiple leafs on exception handling doesn't mean we won't have them in the future. This format allows easy scaling of the test cases as the API surface area and behaviour patterns increase in scope